tests/fixtures/multifile_project/subconfig_2.yml

```diff
@@ -1,3 +1,4 @@
+# Top-level comment
```

Comment:
Is it worth adding comments to the array elements of the subfile, to make sure comments don't go missing when arrays are merged in the single-dict representation?

Reply:
> Is it worth adding comments to the array elements of the subfile, to make sure comments don't go missing when arrays are merged in the single-dict representation?

@kapayne good call. I'll add that to the fixture 👍

Reply:
K, so I found that this was actually a real good test case, because it breaks the order in the second file, since the order itself is just copied from the main file. Gotta fix that.